.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+	"parserOptions": {
```

Comment:
Line 2 and line 3 have tab character and space characters. maybe check soft tabs vs hard tab settings

Reply:
We copy pasted this over from the ES Lint site, so possible we shot ourselves in the foot here